README.md

```diff
@@ -82,12 +92,12 @@ Now that your app is configured with login.gov's sandbox as a Service Provider,
     end
   end
   ```
-6. Add the callback route to `routes.rb`
+7. Add the callback route to `routes.rb`
   ```ruby
   get '/auth/login_dot_gov/callback' => 'users/omniauth#callback'
   ```
 
-7. Start your application and send a `POST` request to: `/auth/login_dot_gov` (eg. http://localhost:3000/auth/login_dot_gov) to initiate authentication with login.gov!
+8. Start your application and send a `POST` request to: `/auth/login_dot_gov` (eg. http://localhost:3000/auth/login_dot_gov) to initiate authentication with login.gov!
```

Comment:
If we're going to renumber these, WDYT about making them all "1"? Since the HTML translation for the pretty README will give them correct numbers anyways?
```suggestion
1. Start your application and send a `POST` request to: `/auth/login_dot_gov` (eg. http://localhost:3000/auth/login_dot_gov) to initiate authentication with login.gov!
```

Reply:
I think that's a great idea! I did a quick attempt before submitting this PR, though, and the current formatting broke the numbering and didn't want to futz with it so reverted.